core-js/src/main/javascript/cdf-legacy/queries/coreQueries.js

```diff
@@ -39,12 +39,7 @@
     deepProperties: ['defaults', 'interfaces'],
     defaults: {
       successCallback: function() {
-        Dashboards.log('Query callback not defined. Override.');
-      },
-      errorCallback: function() {
```

Comment:
Wouldn't it be safer to keep this one and get rid of the other one ?
That way we don't lose the defensive logic we have here.